### DISCUSSION TITLE
Prefixing Class Names in PrimeReact to Resolve CSS Conflicts

### DISCUSSION THREAD

**Discussion by krpdani** [primefaces/community#138] [PrimeReact]
Hello,

I am currently integrating PrimeReact into an existing JSF project which already utilizes PrimeFaces along with its associated CSS. I have encountered an issue where some CSS classes from the old project are conflicting with those from the new PrimeReact integration, given that the new project is nested within the old project.

Is there a method to add a prefix to all PrimeReact class names, or any other workaround to resolve this CSS conflict?

Thank you for your assistance.

Best regards, Danijel

**Comment by mertsincan** [comment]
Hello,

Unfortunately, we do not have such a feature. Can you try to solve it with Find and Replace? Of course, this will be a little annoying. Sorry!
